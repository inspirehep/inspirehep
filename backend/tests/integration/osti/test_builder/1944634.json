{
    "core":false,
    "dois":[
       {
          "value":"10.2172/1827839"
       }
    ],
    "self":{
       "$ref":"https://inspirehep.net/api/literature/1944634"
    },
    "urls":[
       {
          "value":"https://lss.fnal.gov/archive/2021/slides/fermilab-slides-21-054-td.pdf",
          "description":"Fermilab Library Server"
       }
    ],
    "titles":[
       {
          "title":"A large mK platform at Fermilab for quantum computing applications",
          "source":"submitter"
       }
    ],
    "$schema":"https://inspirehep.net/schemas/records/hep.json",
    "authors":[
       {
          "record":{
             "$ref":"https://inspirehep.net/api/authors/1037076"
          },
          "full_name":"Hollister, Matthew I.",
          "affiliations":[
             {
                "value":"Fermilab",
                "record":{
                   "$ref":"https://inspirehep.net/api/institutions/902796"
                }
             }
          ]
       },
       {
          "record":{
             "$ref":"https://inspirehep.net/api/authors/1674023"
          },
          "full_name":"Dhuley, Ram C.",
          "affiliations":[
             {
                "value":"Fermilab",
                "record":{
                   "$ref":"https://inspirehep.net/api/institutions/902796"
                }
             }
          ]
       },
       {
          "record":{
             "$ref":"https://inspirehep.net/api/authors/1500452"
          },
          "full_name":"Tatkowski, Grzegorz",
          "affiliations":[
             {
                "value":"Fermilab",
                "record":{
                   "$ref":"https://inspirehep.net/api/institutions/902796"
                }
             }
          ]
       }
    ],
    "curated":false,
    "documents":[
       {
          "key":"c2b83778ad1534879773dd40ff601972",
          "url":"https://inspirehep.net/files/c2b83778ad1534879773dd40ff601972",
          "filename":"fermilab-slides-21-054-td.pdf",
          "fulltext":true,
          "description":"Fulltext",
          "original_url":"https://lss.fnal.gov/archive/2021/slides/fermilab-slides-21-054-td.pdf"
       }
    ],
    "_collections":[
       "Fermilab"
    ],
    "document_type":[
       "article"
    ],
    "control_number":1944634,
    "report_numbers":[
       {
          "value":"FERMILAB-SLIDES-21-054-TD",
          "source":"submitter"
       }
    ],
    "publication_info":[
       {
          "cnum":"C21-07-19.3",
          "conference_record":{
             "$ref":"https://inspirehep.net/api/conferences/1919279"
          }
       }
    ],
    "acquisition_source":{
       "email":"jtrombet@fnal.gov",
       "orcid":"0000-0002-9381-2678",
       "method":"submitter",
       "source":"submitter",
       "datetime":"2021-10-12T15:09:33.257876",
       "internal_uid":73552,
       "submission_number":"3410045"
    },
    "inspire_categories":[
       {
          "term":"Computing"
       }
    ],
    "external_system_identifiers":[
       {
          "value":"1827839",
          "schema":"OSTI"
       }
    ]
 }