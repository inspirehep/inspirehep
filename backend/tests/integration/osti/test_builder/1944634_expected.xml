<?xml version="1.0" encoding="UTF-8"?>
<records>
   <record>
      <new />
      <site_input_code>FNAL</site_input_code>
      <product_type product_subtype="R">CO</product_type>
      <access_limitation>
         <unl />
      </access_limitation>
      <site_url>https://lss.fnal.gov/archive/2021/slides/fermilab-slides-21-054-td.pdf</site_url>
      <title>A large mK platform at Fermilab for quantum computing applications</title>
      <authors>
         <authors_detail>
            <first_name>Matthew</first_name>
            <middle_name>I.</middle_name>
            <last_name>Hollister</last_name>
            <affiliation>Fermilab</affiliation>
            <private_email />
            <orcid_id />
         </authors_detail>
         <authors_detail>
            <first_name>Ram</first_name>
            <middle_name>C.</middle_name>
            <last_name>Dhuley</last_name>
            <affiliation>Fermilab</affiliation>
            <private_email />
            <orcid_id>0000-0001-6669-9772</orcid_id>
         </authors_detail>
         <authors_detail>
            <first_name>Grzegorz</first_name>
            <middle_name />
            <last_name>Tatkowski</last_name>
            <affiliation>Fermilab</affiliation>
            <private_email />
            <orcid_id />
         </authors_detail>
      </authors>
      <contributor_organizations />
      <report_nos>FERMILAB-SLIDES-21-054-TD</report_nos>
      <doe_contract_nos>AC02-07CH11359</doe_contract_nos>
      <site_input_code>FNAL</site_input_code>
      <country_publication_code>US</country_publication_code>
      <sponsor_org>USDOE Office of Science (SC), High Energy Physics (HEP) (SC-25)</sponsor_org>
      <released_by>Heath O'Connell</released_by>
      <released_by_email>hoc@fnal.gov</released_by_email>
      <released_by_phone>630-840-6017</released_by_phone>
      <medium_code>ED</medium_code>
      <file_format>PDF/A</file_format>
      <description />
      <originating_research_org>Fermi National Accelerator Laboratory (FNAL), Batavia, IL (United States)</originating_research_org>
      <journal_name />
      <journal_volume />
      <journal_issue />
      <product_size />
      <doi />
      <conference_information />
      <other_identifying_nos>oai:inspirehep.net:1944634</other_identifying_nos>
      <publication_date>1900</publication_date>
      <language>English</language>
      <subject_category_code />
      <released_date>10/28/2021</released_date>
   </record>
</records>